app/assets/stylesheets/core/_variables.scss

```diff
@@ -41,7 +41,7 @@ $shaleLighter:    #D4D9DB;
 $shaleLightest:   #F9FBFB;
 
 $brandGreen:      #29BD31;
-$brandHighlight:	#B8E986;
+$brandHighlight:	#B8E98699;
```

Comment:
this seems like a typo

Reply:
the last 2 digits in a 8 digit hex code determine transparency.

#B8E98699 = rgba(184, 233, 134, 0.6)

Reply:
well TIL